pinot-core/src/main/java/com/linkedin/pinot/core/segment/creator/impl/fwd/SingleValueVarByteRawIndexCreator.java

```diff
@@ -44,6 +43,19 @@ public void index(int docId, String valueToIndex)
   }
 
   @Override
+  public void index(int docId, Object valueToIndex)
+      throws IOException {
+    if (valueToIndex instanceof String) {
+      _indexWriter.setString(docId, (String) valueToIndex);
+    } else if (valueToIndex instanceof byte[]) {
+      _indexWriter.setBytes(docId, (byte[]) valueToIndex);
```

Comment:
Not right now. This is in anticipation for supporting columns of type byte[] (blobs).